docs/media-release/index.html

```diff
@@ -0,0 +1,198 @@
+						</div>
+					</a>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<!-- MAIN NAVIGATION -->
+	<nav class="au-main-nav au-main-nav--dark" id="mainmenu" aria-label="main">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+					<div class="au-main-nav__content" id="main-nav-default-dark">
+						<button
+							aria-controls="main-nav-default-dark"
+							class="au-main-nav__toggle au-main-nav__toggle--open"
+							onClick="return AU.mainNav.Toggle( this )">
+							Menu
+						</button>
+						<div class="au-main-nav__menu">
+							<div class="au-main-nav__menu-inner">
+								<div class="au-main-nav__focus-trap-top"></div>
+								<button
+									aria-controls="main-nav-default-dark"
+									class="au-main-nav__toggle au-main-nav__toggle--close"
+									onClick="return AU.mainNav.Toggle( this )">
+									Close
+								</button>
+								<ul class="au-link-list">
+									<li><a href="../">Home</a></li>
+									<li><a href="../basic">Basic page</a></li>
+									<li><a href="../form/">Form page</a></li>
+									<li class="active"><a href="#">Media release</a></li>
+									<li><a href="#">Distinct from eachother</a></li>
+								</ul>
+								<div class="au-main-nav__focus-trap-bottom"></div>
+							</div>
+						</div>
+						<div class="au-main-nav__overlay" aria-controls="main-nav-default-dark" onClick="return AU.mainNav.Toggle( this )"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+
+	<!-- MAIN BODY -->
+	<main id="content">
+		<!--CONTENT-->
+		<div class="au-body container-fluid">
+			<div class="row">
+			<article class="col-md-7 media-release">
+				<h1>Third release of the Trusted Digital Identity Framework</h1>
+				<p class="media-release__caption">
+					We’ve released the third part of the Trusted Digital Identity Framework (TDIF). TDIF sits across all elements of the digital identity program to ensure a consistent approach to usability, accessibility, privacy protection, security and more.
+				</p>
+				<div class="media-release__metadata">
+					<span>Published 10 April 2019</span>
+					<span>Digital Transformation Agency</span>
+					<span>John Smith</span>
+				</div>
+
+				<div class="media-release__article">
+					<p>Taking people on the journey through regular consultation is an important part of how we’re developing the TDIF. At this stage, we have heard from:</p>
+					<ul>
+						<li>federal, state and territory governments</li>
+						<li>privacy advocates</li>
+						<li>digital identity experts</li>
+						<li>the financial sector</li>
+						<li>industry groups</li>
+						<li>standards bodies</li>
+						<li>vendors</li>
+						<li>users and members of the public</li>
+					</ul>
+					<p>Following the consultation period in January, we’re releasing part 3 of the TDIF, which includes 3 new documents and updates to 9 existing documents.</p>
+					<figure class="media-release__figure">
```

Comment:
Do we have other examples where we're using the `figure` element?

Reply:
nope, just thought I'd try it out here